__tests__/app.test.js

```diff
@@ -47,6 +47,56 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api/articles/:article_id/comments", () => {
+  it("GET:200 returns an array of the correct length", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.comments)).toBe(true);
+        expect(body.comments).toHaveLength(11);
+      });
+  });
+
+  it("GET:200 returns elements that each have the correct properties and sorted by most recent first", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.comments.forEach((comment, index) => {
+          expect(comment).toContainKeys([
+            "comment_id",
+            "votes",
+            "created_at",
+            "author",
+            "body",
+            "article_id",
+          ]);
+        });
+
+        expect(body.comments).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+
+  it("GET:404 responds with 'not found' when passed an article_id that doesn't exist", () => {
+    return request(app)
+      .get("/api/articles/14/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("not found");
+      });
+  });
+
+  it("GET:400 responds with 'bad request' when passed an invalid article_id", () => {
+    return request(app)
+      .get("/api/articles/chickens/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
+      });
```

Comment:
Cool, what happens if we ask for comments on article 2? It currently doesn't have any comments, what status code would that be?